sklearn/datasets/olivetti_faces.py

```diff
@@ -64,11 +64,10 @@ def fetch_olivetti_faces(data_home=None, shuffle=False, random_state=0,
         If True the order of the dataset is shuffled to avoid having
         images of the same person grouped.
 
-    random_state : int, RandomState instance or None, optional (default=0)
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState instance or None (default)
```

Comment:
`default` -> `default = 0`

Reply:
Correct me if I'm wrong here but isn't the default `None`, which uses the global random state from `numpy.random`? Which is distinct from `random_state=0` which would set the seed to zero.

Reply:
Fixed in latest commit. Thanks for pointing that out. Apologies for the confusion.